README.md

```diff
@@ -40,14 +40,24 @@ You will need to register your service provider with our sandbox environment usi
 
 Now that your app is configured with login.gov's sandbox as a Service Provider, you can begin configuring your local application.
 
-1. Copy your private key to your application's `config` directory (eg. `config/private.pem`)
+1. Make the private key available to your application. This can be done by running `bin/rails credentials:edit` and adding the following:
+  ```yaml
+  login_pem: |-
+    -----BEGIN PRIVATE KEY-----
+    <PRIVATEKEY>
+    -----END PRIVATE KEY-----
+  ```
 2. Add this gem to the Gemfile:
   ```ruby
   gem 'omniauth_login_dot_gov', :git => 'https://github.com/18f/omniauth_login_dot_gov.git',
                                 :branch => 'main'
   ```
-3. Install this gem and dependencies with `bundle install`
-4. Now, configure the Omniauth middleware with an initializer:
+3. Additionally, add the Omniauth CSRF protection gem to the Gemfile to address the CVE referenced above:
+  ```ruby
+  gem 'omniauth-rails_csrf_protection'
```

Comment:
Do you think it would be harder to skim past if we included both gems in the same step? Just an idea, I don't feel that strongly about it

Reply:
I'm up for either, wasn't sure what the original intention was of putting the CVE in a callout so broke them into separate steps in case that separation and description was useful. But practically I'd expect people just to copy whatever into their Gemfile

Reply:
So since we depend on ominauth, we transitively include the CVE in this project. However, due to the quirky nature of the CVE we can't address it directly so the point of the notice was to have clear instructions for how to address it.

Reply:
Makes sense! I think I'd prefer to leave this called out explicitly.